client/src/api.py

```diff
@@ -57,6 +57,15 @@ def create_app(config: Configuration, tnc_pool: TNCPool, sids_relay: SIDSRelay)
     def get_sids_status():
         return jsonify(sids_relay.get_status()), 200
 
+    @app.route("/api/sids/toggle", methods=["POST"])
+    def toggle_relay():
+        some_json = request.get_json()
```

Comment:
Why is this called "some_json"? This isn't really descriptive of what this json really holds within. I would strongly suggest thinking of a better name.